app/src/main/java/com/heig/atmanager/TodoFeedAdapter.java

```diff
@@ -82,6 +85,9 @@ public void onClick(View view) {
                 holder.expandBtn.setVisibility(View.VISIBLE);
             }
         });
+
+        // Favorite
+        holder.favoriteIcon.setVisibility(todos.get(position).isFavorite() ? View.VISIBLE : View.GONE);
```

Comment:
Show favorite first?

Reply:
I'm putting it as a new issue since this PR was just for the icon